Changes

```diff
@@ -74,8 +74,8 @@ Working version
   (Gabriel Scherer, review by Armaël Guéneau)
 
 * #9207: fix ocamlyacc to work correctly with up to 255 entry points
```

Comment:
The fix isn’t breaking, so this wants to be `-` (not near my laptop, or I’d just push it!)

Reply:
Well spotted, thanks.  Corrected in 584b1429e